<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VisualizationWindow</class>
 <widget class="QWidget" name="VisualizationWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>834</width>
    <height>459</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ալգորիթմի վիզուալիզացիա</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #f9f9f9;</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="AnimationWidget" name="animationArea">
     <property name="styleSheet">
      <string notr="true">border: 0px;
background-color: white;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="animationPlayer" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QSlider" name="animationSlider">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="animationButtons" native="true">
        <property name="styleSheet">
         <string notr="true">QPushButton {
	background-color: white;
}

QPushButton::hover {
	background-color: #f9f9f9;
}

QPushButton:pressed {
    background-color: #f5f5f5;
}
</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="buttonsLayout" stretch="0,13,5,5,5,6,0,3,3">
           <property name="spacing">
            <number>10</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QPushButton" name="regenerateButton">
             <property name="minimumSize">
              <size>
               <width>50</width>
               <height>50</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>50</width>
               <height>50</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">border: 2px solid black;</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::MinimumExpanding</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>60</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="backwardButton">
             <property name="minimumSize">
              <size>
               <width>50</width>
               <height>50</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>50</width>
               <height>50</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">border: 2px solid black;</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="playButton">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>50</width>
               <height>50</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>50</width>
               <height>50</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">border: 2px solid black;</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="forwardButton">
             <property name="minimumSize">
              <size>
               <width>50</width>
               <height>50</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>50</width>
               <height>50</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">border: 2px solid black;</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="changeSpeedButton">
             <property name="styleSheet">
              <string notr="true">QPushButton {
    background-color: black;
    color: white;
    border: 2px solid white;
    border-radius: 10px;
    padding: 10px;
    font-weight: bold;
    text-align: center;
    outline: none;
}

QPushButton:hover {
    background-color: #333333; 
    border-color: #f2f2f2;
    text-decoration: none;
}

QPushButton:pressed {
    background-color: #1a1a1a; 
    border-color: #e6e6e6; 
}
</string>
             </property>
             <property name="text">
              <string>Փոխել արագությունը</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="saveButton">
             <property name="styleSheet">
              <string notr="true">QPushButton {
    background-color: black;
    color: white;
    border: 2px solid white;
    border-radius: 10px;
    padding: 10px;
    font-weight: bold;
    text-align: center;
    outline: none;
}

QPushButton:hover {
    background-color: #333333; 
    border-color: #f2f2f2;
    text-decoration: none;
}

QPushButton:pressed {
    background-color: #1a1a1a; 
    border-color: #e6e6e6; 
}
</string>
             </property>
             <property name="text">
              <string>Պահպանել</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AnimationWidget</class>
   <extends>QGraphicsView</extends>
   <header>animationwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
